runtime/altair/src/liquidity_pools.rs

```diff
@@ -1,99 +0,0 @@
-// Copyright 2023 Centrifuge Foundation (centrifuge.io).
```

Comment:
Why are these pallets removed?

Reply:
unused config, right?

Reply:
Ah, that's right! I forgot to remove those files after migrating them 🤦🏻‍♂️. Thanks